static/images/features/test-analytics-gha-workflow-ex.svg

```diff
@@ -0,0 +1,27 @@
32.667 381.599 232.785 381.485 232.886 381.353C232.991 381.216 233.077 381.068 233.146 380.908V375.604H234.417V383H233.269ZM236.796 385.844V375.604H237.951L238.013 376.41C238.113 376.283 238.222 376.169 238.341 376.068C238.459 375.964 238.587 375.875 238.724 375.802C238.915 375.692 239.127 375.61 239.359 375.556C239.596 375.496 239.849 375.467 240.118 375.467C240.592 375.467 241.011 375.56 241.376 375.747C241.741 375.934 242.046 376.194 242.292 376.526C242.538 376.859 242.725 377.258 242.853 377.723C242.98 378.183 243.044 378.689 243.044 379.24V379.384C243.044 379.912 242.98 380.407 242.853 380.867C242.725 381.323 242.538 381.719 242.292 382.057C242.046 382.394 241.741 382.658 241.376 382.85C241.016 383.041 240.604 383.137 240.139 383.137C239.856 383.137 239.592 383.107 239.346 383.048C239.104 382.989 238.885 382.904 238.689 382.795C238.571 382.727 238.459 382.651 238.354 382.569C238.25 382.483 238.152 382.389 238.061 382.289V385.844H236.796ZM241.779 379.24C241.779 378.88 241.738 378.538 241.656 378.215C241.579 377.887 241.458 377.597 241.294 377.347C241.125 377.101 240.913 376.905 240.658 376.759C240.403 376.613 240.098 376.54 239.742 376.54C239.546 376.54 239.364 376.563 239.195 376.608C239.031 376.654 238.883 376.72 238.751 376.807C238.605 376.893 238.475 377 238.361 377.128C238.247 377.256 238.147 377.397 238.061 377.552V381.113C238.138 381.25 238.227 381.378 238.327 381.496C238.427 381.615 238.541 381.715 238.669 381.797C238.815 381.893 238.977 381.968 239.154 382.022C239.337 382.077 239.537 382.104 239.756 382.104C240.107 382.104 240.41 382.032 240.665 381.886C240.92 381.735 241.13 381.535 241.294 381.284C241.458 381.034 241.579 380.744 241.656 380.416C241.738 380.088 241.779 379.744 241.779 379.384V379.24ZM250.741 379.165H245.512V378.126H250.741V379.165ZM253.612 385.844V375.604H254.768L254.829 376.41C254.929 376.283 255.039 376.169 255.157 376.068C255.276 375.964 255.403 375.875 255.54 375.802C255.731 375.692 255.943 375.61 256.176 375.556C256.413 375.496 256.666 375.467 256.935 375.467C257.409 375.467 257.828 375.56 258.192 375.747C258.557 375.934 258.862 376.194 259.108 376.526C259.354 376.859 259.541 377.258 259.669 377.723C259.797 378.183 259.86 378.689 259.86 379.24V379.384C259.86 379.912 259.797 380.407 259.669 380.867C259.541 381.323 259.354 381.719 259.108 382.057C258.862 382.394 258.557 382.658 258.192 382.85C257.832 383.041 257.42 383.137 256.955 383.137C256.673 383.137 256.408 383.107 256.162 383.048C255.921 382.989 255.702 382.904 255.506 382.795C255.387 382.727 255.276 382.651 255.171 382.569C255.066 382.483 254.968 382.389 254.877 382.289V385.844H253.612ZM258.596 379.24C258.596 378.88 258.555 378.538 258.473 378.215C258.395 377.887 258.274 377.597 258.11 377.347C257.942 377.101 257.73 376.905 257.475 376.759C257.219 376.613 256.914 376.54 256.559 376.54C256.363 376.54 256.18 376.563 256.012 376.608C255.848 376.654 255.7 376.72 255.567 376.807C255.422 376.893 255.292 377 255.178 377.128C255.064 377.256 254.964 377.397 254.877 
```

Comment:
There is a lot of unnecessary precision here (you typically won't ever need more than 2 decimals). Mind running this through https://jakearchibald.github.io/svgomg/?